server/pages/index.js

```diff
@@ -119,7 +122,19 @@ function getCurrentCommitShortChecksum() {
 	}
 }
 
-function getDefaultContext( request ) {
+function getFileContentsSafely( filepath ) {
+	return new Promise( function( resolve ) {
+		fs.readFile( filepath, function( err, content ) {
```

Comment:
Oh, duh, totally. I somehow thought that this was only for build time. Async is definitely the right thing to do.